pyproject.toml

```diff
@@ -71,3 +71,6 @@ module = [
     "streamlit_autorefresh.*",
 ]
 ignore_missing_imports = true
+
+[tool.bandit]
+exclude_dirs = ["tests"]
```

Comment:
Do we still need this? I had bandit from ruff configured such that it wasn't annoying in tests

Reply:
removed